nuxeo-wopi-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-wopi/nuxeo-wopi.html

```diff
@@ -0,0 +1,24 @@
+<!--
+(C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and contributors.
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+​
+    http://www.apache.org/licenses/LICENSE-2.0
+​
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Contributors:
+  Thomas Roger <troger@nuxeo.com>
+-->
+
+<link rel="import" href="nuxeo-wopi-link.html">
+
+<nuxeo-slot-content name="wopiLink" slot="BLOB_ACTIONS">
+  <template>
+    <nuxeo-wopi-link document="[[document]]" blob="[[blob]]" xpath="[[xpath]]"></nuxeo-wopi-link>
```

Comment:
I would also try to move the `_isAvailable` filter logic here by surrounding your action with a `nuxeo-filter` and use the `expression` attribute to compute the availability. The goal is to let end user override the slot contrib to extend/restrict availability

Reply:
Will be done when using the wopi enricher, after https://jira.nuxeo.com/browse/NXP-25512.